go.mod

```diff
@@ -22,8 +22,8 @@ require (
 	k8s.io/code-generator v0.22.5
 	knative.dev/caching v0.0.0-20220302153644-5a6403ef2de5
 	knative.dev/eventing v0.30.0
-	knative.dev/hack v0.0.0-20220224013837-e1785985d364
-	knative.dev/pkg v0.0.0-20220301181942-2fdd5f232e77
+	knative.dev/hack v0.0.0-20220302134343-0687d6e8cd79
```

Comment:
the hash for the 1.3 release for hack should be [e178598](https://github.com/knative/hack/commit/e1785985d364ff2ac43a33360a26c2027a02b1cf)

Reply:
This is automatically generated by running the script: ./hack/update-deps.sh --upgrade --release v0.30, so it should be fine.